### PR TITLE
Update vanilla-lazyload

### PR DESCRIPTION
**Pull request by verlok**

### CODE REVIEW

File: ajax/libs/vanilla-lazyload/package.json [hunk 1]
```diff
@@ -1,24 +1,34 @@
 {
   "name": "vanilla-lazyload",
   "filename": "lazyload.min.js",
-  "version": "12.3.0",
-  "description": "LazyLoad is a fast, lightweight and flexible script for loading images only when they're about to enter the viewport of a scrollable area, with an excellent support to the progressive JPEG image format. Typescript module definition is available.",
+  "version": "15.1.1",
```
Comment (MattIPv4) [inline ajax/libs/vanilla-lazyload/package.json:4]:
```suggestion
  "version": "12.3.0",
```
Version should not be changed, the auto-update bot will do this.

File: ajax/libs/vanilla-lazyload/package.json [hunk 2]
```diff
@@ -1,24 +1,34 @@
...
-  "authors": [
-    "Andrea",
-    "verlok",
-    "Verlicchi"
-  ],
+  "author": {
+        "name": "Andrea \"verlok\" Verlicchi",
+        "email": "andrea.verlicchi@gmail.com",
+        "url": "https://www.andreaverlicchi.eu"
+    },
```
Comment (MattIPv4) [inline ajax/libs/vanilla-lazyload/package.json:15]:
```suggestion
  "author": {
    "name": "Andrea \"verlok\" Verlicchi",
    "email": "andrea.verlicchi@gmail.com",
    "url": "https://www.andreaverlicchi.eu"
  },
```
Please use the correct indent of two spaces.

File: ajax/libs/vanilla-lazyload/package.json [hunk 3]
```diff
@@ -1,24 +1,34 @@
...
+        "srcset",
+        "SEO",
+        "intersectionObserver",
+        "sizes",
+        "progressive",
+        "performance",
+        "perfmatters",
+        "async",
+        "no-jquery"
+    ],
```
Comment (MattIPv4) [inline ajax/libs/vanilla-lazyload/package.json:31]:
```suggestion
    "lazyload",
    "vanilla",
    "responsive",
    "images",
    "picture",
    "srcset",
    "SEO",
    "intersectionObserver",
    "sizes",
    "progressive",
    "performance",
    "perfmatters",
    "async",
    "no-jquery"
  ],
```
Please use the correct indent of two spaces.

**Comment by verlok** [comment]
Thanks. I've committed your suggestions.

**Pull request merged**